js/src/wp-seo-admin-global.js

```diff
@@ -128,3 +128,55 @@
 	window.wpseoMakeDismissible = wpseoMakeDismissible;
 	window.wpseoDismissLink = wpseoDismissLink;
 }());
+
+(function() {
```

Comment:
Using `(function($) {})(jQuery);` would make line 135 redundant and is best practice, as far as I know.